tests/utils/helper_funcs.py

```diff
@@ -0,0 +1,71 @@
+import tests.factories as factories
+
+
+def delete_records(session, records):
+    for record in records:
+        session.delete(record)
+    session.commit()
+    return
+
+
+def insert_record(session):
```

Comment:
looks fine to me but maybe we can make this a bit more concise by using a for loop (similar to what we do in the ETL). fine to do this in another PR tho

Reply:
@simonarocco actually I'll just address this. need to make some changes to this anyways bc of estimate pins related schema updates